src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2858,6 +2874,21 @@ namespace winrt::TerminalApp::implementation
         _FullscreenChangedHandlers(*this, nullptr);
     }
 
+    void TerminalPage::Maximized(bool newMaximized)
+    {
+        _isMaximized = newMaximized;
+    }
+
+    void TerminalPage::RequestSetMaximized(bool newMaximized)
+    {
+        if (_isMaximized == newMaximized)
+        {
+            return;
+        }
+        _isMaximized = newMaximized;
+        _ChangeMaximizeRequestedHandlers(*this, nullptr);
+    }
```

Comment:
I'd appreciate a comment here saying when to use one over the other. It seems like you we should only be using the first one in `AppLogic` during startup?

Reply:
`Maximized` is also used when the window maximized state is changed otherwise. E.g. if the user clicks the maximize button or uses an OS hotkey to do so. I'll add comments.